.travis.yml

```diff
@@ -1,8 +1,14 @@
 sudo: false
 language: node_js
 node_js:
-  - "4.1.1"
-  - "0.12"
+  - "stable"
+  - "6"
+  - "4"
 before_install:
-  -  git config --global user.email "you@example.com"
-  -  git config --global user.name "Your Name"
+  - git config --global user.email "you@example.com"
+  - git config --global user.name "Your Name"
+install_script:
```

Comment:
Please remove the `install_script` and `script` block. Travis should not include custom directives.